cumulus/tasks/pdr-status-check/tests/index.js

```diff
@@ -61,26 +64,35 @@ test('returns the correct results in the nominal case', (t) => {
     'arn:1': 'RUNNING',
     'arn:2': 'SUCCEEDED',
     'arn:3': 'FAILED',
-    'arn:4': 'ABORTED'
+    'arn:4': 'ABORTED',
+    'arn:7': null
+  };
+  const error = {
+    message: 'Execution Does Not Exist: arn',
```

Comment:
This is always going to set the message to `Execution Does Not Exist: arn`, not the actual execution arn.

Reply:
done